de.nicokimmel.shadowcast-electron.json

```diff
@@ -0,0 +1,25 @@
+{
+  "app-id": "de.nicokimmel.shadowcast-electron",
+  "runtime": "org.electron.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.electron.Sdk",
+  "command": "shadowcast-electron",
+  "finish-args": [
+    "--device=all",
+    "--socket=pulseaudio"
+  ],
+  "modules": [
+    {
+      "name": "shadowcast-electron",
+      "buildsystem": "electron",
```

Comment:
this is no such build system. I guess you didn't try to build locally.

Reply:
Its upsteam now and installed when building.